12_glob_lab/globlab.txt

```diff
@@ -5,25 +5,34 @@ https://www.tocode.co.il/bundles/unix1/lessons/globlab
 
   1. All files whose name starts with a lowercase letter
   Answer: 
-
+ls [a-z]*
   2. All files whose name starts with an uppercase letter
   Answer:
-
+ls [A-Z]*
   3. All files whose name ends in an "a", "e", "i", "o" or "u"
   Answer:
 
+  leo@leo-VirtualBox:~/lab$ ls *.[a,e,i,o,u]
+
   4. All files whose name DOES NOT end in an "a", "e", "i", "o" or "u"
   Answer:
+leo@leo-VirtualBox:~/lab$ ls *.[!a,e,i,o,u]
 
   5. All files with extension ".c" that are in a folder named "src"
   Answer:
-
+ls *.c src/
   6. All files with extension ".c" under any folder
   Answer:
-
+find . -type f -name *.c
 2. Write a command that would delete all files ending with a ".png" under any subdirectory.
 
+find . -type f -name *.png -delete
+
 3. The command "ls -l" can take the name of a file or directory and show details on them. From within "wildcards" folder run "ls -l [A-Z]*". What was printed? Why?
 
+1 hidden files
```

Comment:
all files that begin with *any* letter except of lower case "a" or all files that begin will an upper-case letter depends on your locale, [I try to explain here](https://github.com/tocodeil/unix1-course-examples/issues/15)